Assets/NuGet/Editor/NugetHelper.cs

```diff
@@ -1160,19 +1160,30 @@ private static void WarnIfDotNetAuthenticationIssue(Exception e)
         /// <returns>Stream containing the result.</returns>
         public static Stream RequestUrl(string url, string password, int? timeOut)
         {
+            string packageHost = new Uri(url).Host;
+
             HttpWebRequest getRequest = (HttpWebRequest)WebRequest.Create(url);
             if (timeOut.HasValue)
             {
                 getRequest.Timeout = timeOut.Value;
                 getRequest.ReadWriteTimeout = timeOut.Value;
             }
+
+            if (packageHost.EndsWith("pkgs.visualstudio.com") && password == null)
```

Comment:
Small change.

We should change `password == null` to `string.IsNullOrEmpty(password)`. They might have the password box ticked and nothing provided.

Reply:
Good point! I'll get that updated.